src/game/Entities/Unit.cpp

```diff
@@ -4128,6 +4128,8 @@ void Unit::_UpdateAutoRepeatSpell()
         // cancel wand shoot
         if (m_currentSpells[CURRENT_AUTOREPEAT_SPELL]->m_spellInfo->Category == 351)
             InterruptSpell(CURRENT_AUTOREPEAT_SPELL);
+        else if (getAttackTimer(RANGED_ATTACK) < 500)
+			setAttackTimer(RANGED_ATTACK, 500);
```

Comment:
This line here is also indented with tabs, this project uses 4 white-space characters in place of a tab, like you've correctly done on the line above it.